### ISSUE TITLE
Error page is sometimes displayed incorrectly.

### ISSUE THREAD

**Issue by jamal2362** [Slion/Fulguris#208]
Error page is sometimes displayed incorrectly.
This is how you can reproduce it.

1. Turn off the Internet.
2. Open Fulguris.
3. Open a new tab.
4. Type anything in the search bar at the bottom. (Not the upper one!)
5. Click on Search.
6. See the error...

It may not happen the first time.
Then you have to try again until the error occurs.

**Comment by Slion** [comment]
Yes I know, in some rare situations the theming does not work for some reason. Timing dependent it seems.
